src/index.js

```diff
@@ -2,11 +2,9 @@ const express = require('express');
 const app = express();
 const path = require('path');
 
-app.get('/', (req, res) => {
-    const htmlPath = path.join(__dirname, 'index.html');
+const staticPath = path.join(__dirname, '..', 'resources', 'html');
```

Comment:
Well, the `use path.join for constructing paths` rule can be weakened for string literals. The rule was introduced due to inconsistencies with path separators on different operating systems (`/` on *nix, `\` on Windows). When you use a simple string concatenation (`+`, `+=`, template strings) you can get a path with a mixture of different path separators. `path.join` always replaces `/` with a native path separator in the passed arguments, so it's safe to use `string/literal/paths` when you use `path.join`.